Weather App/UI/HomeVC/HomeCoordinator.swift

```diff
@@ -33,12 +32,17 @@ class HomeCoordinator : Coordinator {
         let vc = HomeViewController()
         let vm = HomeViewModel()
         
-        vc.settingsPressed = {
+        vm.settingsPressed = {
             [weak self] in
             self?.showSettingsVC()
         }
         
-        //ovdje callbackovi za VM
+        vm.searchPressed = {
+            [weak self] in
```

Comment:
Should be in the above line, after the curly brackets.

Reply:
I will remove the comment.